config/config_test.go

```diff
@@ -2136,38 +2136,34 @@ func TestExpandExternalLabels(t *testing.T) {
 	// Cleanup ant TEST env variable that could exist on the system.
 	os.Setenv("TEST", "")
 
-	c, err := LoadFile("testdata/external_labels.good.yml", false, false, log.NewNopLogger())
-	require.NoError(t, err)
-	testutil.RequireEqual(t, labels.FromStrings("bar", "foo", "baz", "foo${TEST}bar", "foo", "${TEST}", "qux", "foo$${TEST}", "xyz", "foo$$bar"), c.GlobalConfig.ExternalLabels)
```

Comment:
Why drop this test?

Reply:
This test still needs addressing. Please make sure you understand what this change is implementing.

Reply:
This test is to verify the parsing when `expand-external-labels` is set to false, but now that `expand-external-labels` is true by default, the result will be the same as other tests in this func, so I think we can just remove it.
if you have a better solution🤔, plz tell me :) thx

Reply:
Sorry, yes that is what I meant. My original comment meant to read "Why *not* drop this test".
Sorry for the confusion...I agree this test should be dropped.